README.md

```diff
@@ -1,36 +1,99 @@
+Here's a [list of features and issues](https://github.com/xmtp/xmtp-react-native/issues) we're still working on. 
 
-# Installation in bare React Native projects
+Is there a feature you need that's not on the list? Please [open an issue](https://github.com/xmtp/xmtp-react-native/issues).
 
-For bare React Native projects, you must ensure that you have [installed and configured the `expo` package](https://docs.expo.dev/bare/installing-expo-modules/) before continuing.
+Or better yet, open a PR and we'll get it reviewed and merged as soon as possible. If you contribute a PR that gets merged into this repo, you'll be eligible to [claim this XMTP contributor POAP](https://www.gitpoap.io/gp/1042)!
 
-### Add the package to your npm dependencies
+To learn more about XMTP and get answers to frequently asked questions, see [FAQ about XMTP](https://xmtp.org/docs/dev-concepts/faq).
+
+![x-red-sm](https://user-images.githubusercontent.com/510695/163488403-1fb37e86-c673-4b48-954e-8460ae4d4b05.png)
+
+## Example apps
+
+This repo provides the following example apps that you can use to experiment and explore implementation details:
 
+- [Android example](example/android)
+
+- [iOS example](example/ios)
+
+To use an example app, run:
+
+```bash
+cd example
+npm install --force
+npm run [ios or android]
 ```
+
+## Install in a managed Expo project
+
+<!--eng: When I run this at the root of the project, I get this prompt: The android and ios projects are malformed, would you like to clear the project files and reinitialize them? › (Y/n) I imagine this is because I'm not running it in a managed Expo project. Just wanted to run it to see what happened. =)-->
+
+```bash
+npx expo prebuild
+```
+
+## Install in bare React Native project
+
+For bare React Native projects, [install and configure the `expo` package](https://docs.expo.dev/bare/installing-expo-modules/) before continuing.
+
+### Add the package to your npm dependencies
+
+<!--eng: Should this command use xmtp-react-native or xmtp-react-native-sdk? When I run this command, I get npm ERR! 404 Not Found - GET https://registry.npmjs.org/xmtp-react-native - Not found. This is because we haven't submitted to npm yet, correct? Will this command work for pre-preview?-->
+
+```bash
 npm install xmtp-react-native-sdk
 ```
 
 ### Configure for iOS
 
-Run `npx pod-install` after installing the npm package.
+After installing the npm package, run:
+
+```bash
+npx pod-install
+```
 
 ### Configure for Android
 
-# Contributing
+<!--eng: Seems weird to have a section to say that there's nothing to do, but thought it might be helpful to explictly call this out. If we leave it out, folx might think the Android info is missing. WDYT?-->
```

Comment:
I think once we're published to npm and can E2E test the installation flow, we'll have more to say about platform specific setup. It's sort of an option question about how this stuff works right now to me.